catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/BrickValueParameterTest.java

```diff
@@ -316,6 +1057,25 @@ private void checkIfBrickShowsSpinnerWithEditTextOverlayWithText(Class brickClas
 				.check(matches(withText(stringResourceId)));
 	}
 
+	private void checkIfBrickAtPositionShowsSpinnerWithEditTextOverlayWithText(Class brickClass, int position, int
+			spinnerResourceId,
+			int stringResourceId) {
+		onData(instanceOf(brickClass)).inAdapterView(BrickPrototypeListMatchers.isBrickPrototypeView())
+				.atPosition(position)
+				.onChildView(withId(spinnerResourceId))
+				.check(matches(withSpinnerText(stringResourceId)));
+	}
+
+	private void checkIfBrickAtPosition2ShowsSpinnerWithEditTextOverlayWithText(Class brickClass, int position, int
```

Comment:
this implies to me that this checks the brick at position TWO ....
also see comment below at 1105